libbeat/docs/release-notes/breaking/breaking-7.7.asciidoc

```diff
@@ -12,9 +12,25 @@
 
 //tag::notable-breaking-changes[]
 
-//[float]
-//==== Breaking change
+[float]
+====  Environment variables can no longer reference other environment variables
 
-//Description
+Environment variables are still supported in the {beats} configuration.
+However, starting in Version 7.7.0, you can no longer reference an environment
+variable that references another environment variable.
+
+For example, if you set `export NAME=${COMPANY_NAME}`, you cannot use
+`name: ${NAME}` in the configuration.
+
+[float]
+==== Docker and kubernetes processors no longer allowed in script processor
+
+Prior to this release, it was possible to use the `add_docker_metadata` and
+`add_kubernetes_metadata` processors in the `script` processor. This was not a
+good practice because it sometimes resulted in memory and file descriptor leaks.
+
+Starting in version 7.7.0, scripts that use these processors will fail. To
+resolve this problem, define the processors in your configuration instead of the
+script.
```

Comment:
@exekias Please have a look. Anything else missing?

Reply:
This looks good to me